### PR TITLE
Financial instruments transformer - Add endpoint to get all FIs APIURLs

### PR DESCRIPTION
**Pull request by iulianp26**

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
The transformer should have a __reload endpoint, to match all of the others.

**Comment by Nesze** [comment]
Yeah, we'll add it (but for the record, I think most of the transformers lack that endpoint).

File: handler_test.go [hunk 1]
```diff
@@ -208,6 +208,70 @@ func TestId_FinancialInstrumentExists_OkStatusAndCorrectFIReturned(t *testing.T)
...
+	req, err := http.NewRequest("GET", "http://fiTransformer", nil)
+	if err != nil {
+		t.Fatalf("Failure in setting up the test request: [%v]", err)
+	}
+
+	w := httptest.NewRecorder()
+
+	h.getFinancialInstruments(w, req)
+
+	if w.Code != 503 {
```
Comment (tosan88) [inline handler_test.go]: Can you please use `require.Equals` instead?

File: service.go [hunk 2]
```diff
@@ -37,6 +39,15 @@ func (fis *fiServiceImpl) IDs() []string {
 	return UUIDs
 }
 
+func (fis *fiServiceImpl) apiUrls() []string {
+	var apiUrls = []string{}
+	for uuid := range fis.financialInstruments {
+		apiUrl := fis.baseUrl + uuid
```
Comment (tosan88) [inline service.go]: I expect constructing the url is the reponsibility of the handler and not the service itself... Wouldn't be better just to return the uuids here? What is your opinion?

**Comment by tosan88** [comment]
@david-smith-ft @Nesze I also think that the lack of `__reload` is not a blocker for this change - I would see it in another PR...

File: service.go (hunk 2 above)
Comment (iulianp26) [reply service.go]: Sounds ok to me. I'll make the change.

**Review by tosan88 — APPROVED** [review]

**Comment by Nesze** [comment]
cc: @nwrigh

**Review by ghost — APPROVED** [review]

**Pull request merged**